.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/aws-cloudformation/cfn-lint
-    rev: "v0.83.1"
```

Comment:
1+y old version, we should look into update to find as as current issues as possible  (in separate ticket)

Reply:
not part of change. I just formatted code base to comply prettier config